### ISSUE TITLE
There's multiple issues with OpenGOAL when running on a non ASCII path

### ISSUE THREAD

**Issue by Adamillo** [open-goal/jak-project#1629]
### Acknowledgements

- [X] I have [searched](https://github.com/open-goal/jak-project/issues?q=is%3Aissue+is%3Aopen+label%3Abug+sort%3Aupdated-desc) for duplicate or closed bug reports
- [X] I understand that I am supposed to provide my own legitimately obtained copy of the game

### Describe the Bug

As I said there's many problems with OpenGOAL when run under a non ASCII. For one, the extractor opens then closes immediately when run in a non ASCII path. Another issue is that OpenGOAL will crash when trying to create a save file on a non ASCII path, so that means that the game is unplayable without debug mode.

### How To Reproduce

On Windows: Run OpenGOAL with a local username that has non ASCII characters.

### Does this problem occur on original hardware or PCSX2?

Not needed, bug is obvious

### Expected Behavior

When running OpenGOAL on a non ASCII path, the extractor should work, and the game shouldn't crash when trying to create a save file

### Operating System

Windows 10

### OpenGOAL Version

v0.1.17

### Game Version

NTSC 1.0 (black label)

### Are you using a high refresh-rate monitor (>60Hz)

No

**Comment by xTVaser** [comment]
v0.1.17 explicitly solved a lot of these problems and I ran the extractor on a unicode path myself.  Can you give specific information about what paths are being used (ISO path / path where you're extracting the release, etc)

**Comment by Adamillo** [comment]
The game path is in the same path where the extractor is located at. At first I tried extracting the game on  C:/Users/вав/Downloads/opengoal-v0.1.17-windows. After that used the path C:/opengoal-v0.1.17-windows there and the extractor started working again. After the extractor finished I tried starting a new game and since the save file tries to create a save on a path which is C:/Users/вав/appdata/roaming it crashes since it's also non ASCII
